slack-api-model-kotlin-extension/src/main/kotlin/com/slack/api/model/kotlin_extension/block/composition/dsl/RichTextObjectDsl.kt

```diff
@@ -0,0 +1,75 @@
+package com.slack.api.model.kotlin_extension.block.composition.dsl
+
+import com.slack.api.model.block.element.RichTextSectionElement.LimitedTextStyle
+import com.slack.api.model.block.element.RichTextSectionElement.TextStyle
+import com.slack.api.model.kotlin_extension.block.BlockLayoutBuilder
+import com.slack.api.model.kotlin_extension.block.element.BroadcastRange
+
+@BlockLayoutBuilder
+interface RichTextObjectDsl {
+
+    /**
+     * @param range The range of the broadcast.
+     * @see <a href="https://api.slack.com/reference/block-kit/blocks#broadcast-element-type">Rich text object documentation</a>
+     */
+    fun broadcast(range: BroadcastRange)
+
+    /**
+     * @param value The hex value for the color.
+     * @see <a href="https://api.slack.com/reference/block-kit/blocks#color-element-type">Rich text object documentation</a>
+     */
+    fun color(value: String)
```

Comment:
same as above

Reply:
Resolved in 1be897c8ce41fb2d32ee1b4cdaa0d647fd7b43c0